### ISSUE TITLE
Optionally install Windows 10-style GTK/QT themes in GUI section of wlinux-setup

### ISSUE THREAD

**Issue by sirredbeard** [WhitewaterFoundry/WLinux#159]
- https://github.com/B00merang-Project/Windows-10
- https://github.com/B00merang-Project/Windows-10-Dark

**Comment by patrick330602** [comment]
Also [A icon theme](https://store.kde.org/p/1156796/) and a [Cursor theme](https://store.kde.org/p/1084938/) would be good.

**Comment by grufwub** [comment]
Working on this here: https://github.com/grufwub/WLinux/tree/win10-theme
Just building latest install.tar.gz and going to test initial commits. At the moment it downloads + unpacks to .themes folder, but am going to add a prompt asking whether the user wants to enable Light / Dark theme then sets as default theme.

**Comment by grufwub** [comment]
Getting it unpacked and installed in the right location is no issue, but having it set the theme is more complex. Also depending on the application may not have much (if any) effect.

We could set it to automatically install to the /usr/share/themes folder for now with the assumption that the user will be installing a full desktop env. so that they can set the theme? I can easily set it so that it also sets the appropriate GDK_THEME, QT_STYLE_OVERRIDE environment variables too for the windows-10 themes, but I haven't been able to confirm that it's actually working yet.

**Comment by grufwub** [comment]
See #260

**Comment by grufwub** [comment]
Merged into WLinux/development, now waiting on #256. Though planned features in future
